Gravity/Gravity.Test.Unit/RsapiDaoDeleteTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
```

Comment:
I do not see this load with the project.  I do see it on disk.  Is that intentional?  I think it is fine to add to project.

Reply:
just let me know if these are supposed to be included in the commit.  If so they should be part of the project.

Reply:
This has been taken care of in the last commit.

Reply:
sorry missed that.